portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
+/*
+ * Displays the selected tab on the homepage
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function openTab(event, tabName) {
```

Comment:
Instead of openTab, I think displayTab describes the action better. What do you think?

Reply:
Definitely, thanks for the suggestion!